code/modules/surgery/organs/lungs.dm

```diff
@@ -316,6 +317,16 @@
 	safe_toxins_min = 16 //We breath THIS!
 	safe_toxins_max = 0
 
+/obj/item/organ/lungs/vox
+	name = "vox lungs"
+	icon_state = "lungs-x"
+	desc = "Pretty dusty."
+	safe_oxygen_min = 0 //We don't breathe this
+	safe_oxygen_max = 1 //This is toxic to us
+	safe_nitro_min = 16 //We breathe THIS!
+	oxy_damage_type = TOX //Oxygen poisons us
+	oxy_damage_multiplier = 10 // BADLY.
```

Comment:
Base level is 1/second, roughly. This increases it to.. 10. I might kick it down to 5, that way it's not as bad, but it still needs to be bad.

Reply:
Actually I will make it 5. That way they can eat without fucking dying. 🤔